docker-compose.yml

```diff
@@ -96,14 +109,33 @@ services:
       - chs-dev
     expose:
       - 2000
+  chs-company-profile-api:
+    container_name: chs-company-profile-api
```

Comment:
If we are removing 'chs' prefix, should it be removed here as well?

Reply:
since that is the name of the integration, I stuck with the name.